scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/ScipBuildTool.scala

```diff
@@ -657,5 +783,9 @@ object ScipBuildTool {
   // canonical URLs will become 404 links. The lsif-java.json file is not
   // supposed to be written by end-users anyways. It's mostly an implementation
   // default for how we support cross-repo navigation with scip-java.
-  val ConfigFileName = "lsif-java.json"
+  val ConfigFileNames = List("scip-java.json", "lsif-java.json")
+  val isIgnoredAnnotationProcessor = Set(
+    "",
```

Comment:
IMO this is confusing - let's replace it with `isEmpty` check at use site instead.